script/build.js

```diff
@@ -395,7 +395,6 @@ if (options.buildtype !== 'development') {
 
 smith.use(redirect({
   '/2015/11/11/why-we-are-designing-in-beta.html': '/2015/11/11/why-we-are-designing-in-beta/',
-  '/disability-benefits/apply/': '/disability-benefits/apply/',
```

Comment:
@tiffanybbrown-va there are more paths under `/disability-benefits/apply-for-disability-benefits/` that need to be redirected:

    /disability-benefits/apply-for-benefits/help/  -> /disability-benefits/apply/help/
    /disability-benefits/apply-for-benefits/one-year/ -> /disability-benefits/apply/one-year/
    /disability-benefits/apply-for-benefits/ways/ -> /disability-benefits/apply/ways/

We could set them each up here individually or we can do a coordinated deploy of vets-website and https://github.com/department-of-veterans-affairs/devops/pull/1627.